examples/model_compress/knowledge_distill/KnowledgeDistill.md

```diff
@@ -0,0 +1,31 @@
+Knowledge Distillation on NNI Compressor
```

Comment:
not suggest to put doc under examples folder

Reply:
you could put it under `docs/en_US/TrialExample/`, and properly reference it in `Compressor/Overview.md`

Reply:
resolved.